### ISSUE TITLE
Relative import issue...

### ISSUE THREAD

**Issue by kif** [serge-sans-paille/pythran#738]
Hi Serge,

Apparently, a relative python import causes pythran to complain with this error. 
```
pythran.syntax.PythranSyntaxError: Specifying a level in an import
```

It would be great to make the exception message more explicit (at least mention it may be related to relative imports which are not supported)

Cheers,

Jerome

**Comment by serge-sans-paille** [comment]
Fixed in #742

**Comment by serge-sans-paille** [comment]
Merged. Thanks for reporting!
